### DISCUSSION TITLE
Site design for fluk3yluk3.com

### DISCUSSION THREAD

**Discussion by fluk3yluk3** [ardacarofficial/links-website#10] [Your Demos]
Hey just writing to say thanks for sharing your links-website project. I have borrowed the code and made a bunch of changes and published my own version for my portfolio/links website. Really appreciate the work you put into build and document this project!

[fluk3yluk3.com](https://fluk3yluk3.com) and source here: [github](https://github.com/fluk3yluk3/fluk3yluk3.github.io)

Cheers,
fluk3yluk3

**Comment by ardacarofficial** [comment]
I love the CSS changes and the background is great! 

I also appreciate your thanks.
